sealedsecret-crd.jsonnet

```diff
@@ -30,6 +30,14 @@ local crd = {
                 type: "string",
                 pattern: "^[^A-Za-z0-9+/=]*$", // base64
               },
+              encryptedData: {
+                patternProperties: {
+                  "^.*$": {
+                    pattern: "^[^A-Za-z0-9+/=]*$", // base64
```

Comment:
I think we should keep this pattern the same as above, so either we remove the `^` from both or leave it as it is for now and you can fix both in a later PR.

Reply:
Yep, that makes sense.